sympy/codegen/ast.py

```diff
@@ -54,7 +54,6 @@
        |        |--->NoneToken
        |
        |--->Statement
-       |--->Return
 
```

Comment:
Is that the correct line removed?

Reply:
No, it is not, it should be the the line above.

Reply:
Is it right now? I don't see the problem.

Reply:
Previously, I had removed `Return` rather than `Statement`, which I reversed in the second commit. Then a different PR got merged which placed `Return` under token and caused merge conflicts which I resolved in the third commit. Hopefully this doesn't cause confusion in the future.